cmd/aries-js-worker/main.go

```diff
@@ -38,6 +37,7 @@ const (
 	ariesCommandPkg  = "aries"
 	ariesStartFn     = "Start"
 	ariesStopFn      = "Stop"
+	workers          = 9
```

Comment:
Why 9?

Reply:
eg., I think JS in worker only has one thread.

Reply:
`why 9?` just in case :) I can set it to `3`

Reply:
why 3

Reply:
i assume 3 is the number of workers that rids us of the problem? if so, great.

Reply:
let me check it

Reply:
Somehow I thought it might be 2 :)

Reply:
@troyronda works with `2` as well